### ISSUE TITLE
Failed to guess the relative scripts/styles paths in WP subfolder installs

### ISSUE THREAD

**Issue by nash-ye** [xwp/wp-dependency-minification#34]
This is something I notice while I am working on PR #31 , WordPress register most of their scripts/styles that included in wp-includes folder with relative paths.

So a script like comment-reply can't be optimized successfully, when I dig more into this it seems that our way of guessing the URL/Path is wrong.

see this codes:

``` php
        $has_host = (bool) parse_url( $src, PHP_URL_HOST );
        if ( ! $has_host ) {
          $src = 'http://' . $host_domain . $src;
        } 
```

and

``` php
        // First attempt to get the file from the filesystem
        $contents = false;
        $is_self_hosted = self::is_self_hosted_src( $src );
        if ( $is_self_hosted ) {
          $src_abspath = ABSPATH . parse_url( $src, PHP_URL_PATH );
          $contents = file_get_contents( $src_abspath );
        } 
```

**Comment by nash-ye** [comment]
I think this issue is ready to go!

**Comment by westonruter** [comment]
@nash-ye awesome! Could you open a PR to the `x-team/develop` branch?

**Comment by nash-ye** [comment]
@westonruter done :+1:

**Comment by westonruter** [comment]
Released in v0.9.7: http://wordpress.org/plugins/dependency-minification/
